tests/pytype_test.py

```diff
@@ -70,7 +70,7 @@ def pytype_test(args):
         print("Cannot run pytd. Did you install pytype?")
         return 0, 0
 
-    wanted = re.compile(r"stdlib/(2|2\.7|2and3)/.*\.pyi$")
+    wanted = re.compile(r"stdlib/(2|2\.7|2and3)/.*\.pyi?$")
```

Comment:
If you *do* run pytype, you may have to run it as `TYPESHED_HOME=/some/path/ pytype` in order to point it at the typeshed version you're testing.